server/somesockets.py

```diff
@@ -1,47 +1,81 @@
-from flask import Flask, render_template
+from flask import Flask, render_template, request
 from flask_socketio import SocketIO, send, emit
+from secrets import SOCKETIO_SECRET_KEY
+from models import Session
+from flask_mongoengine import MongoEngine
 
+#=====================================================
+# Global vars
+#=====================================================
 app = Flask(__name__)
-app.config['SECRET_KEY'] = 'secret!'
+app.config['SECRET_KEY'] = SOCKETIO_SECRET_KEY
+app.config.from_object('config')
+
+db = MongoEngine(app)
+
 socketio = SocketIO(app)
+open_rooms = dict()
+bijection = dict()
```

Comment:
What does bijection hold?

Reply:
thats gonna change it was an idea i had but it's not going to be used